msgid ""
msgstr ""
"Project-Id-Version: CaptchaSolver\n"
"POT-Creation-Date: 2017-08-30 19:24+0800\n"
"PO-Revision-Date: 2017-08-30 19:32+0800\n"
"Last-Translator: Kvark <kvark128@yandex.ru>\n"
"Language-Team: \n"
"Language: ru_RU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.6.11\n"
"X-Poedit-Basepath: .\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Poedit-SearchPath-0: ..\\..\\..\\globalPlugins\n"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:20
msgid "Captcha Solver"
msgstr "Captcha Solver"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:37
msgid ""
"Captcha successfully sent to the recognition. You will be notified when the "
"result will be ready"
msgstr ""
"Капча успешно отправлена на распознавание. Вы будете уведомлены, когда "
"результат будет готов"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:48
msgid "Captcha solved successfully! The result copied to the clipboard"
msgstr "Капча успешно распознана! Результат помещён в буфер обмена"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:55
#, python-brace-format
msgid "Balance: {balance:.2f} rubles"
msgstr "Баланс: {balance:.2f} рублей"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:62
msgid "Error: {}"
msgstr "Ошибка: {}"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:82
#, python-brace-format
msgid "Size: {0} X {1} pixels"
msgstr "Размер: {0} на {1} пикселей"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:89
msgid "Starts the recognition process"
msgstr "Начать процесс распознавания"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:93
msgid "Report account balance"
msgstr "Сообщить баланс аккаунта"

#: ..\..\..\globalPlugins/captchaSolver/__init__.py:97
msgid "Show the settings dialog"
msgstr "Показать диалог настроек"

#: ..\..\..\globalPlugins/captchaSolver/_config.py:30
msgid "Error saving settings"
msgstr "Ошибка при сохранении настроек"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:10
msgid "Captcha Solver Settings"
msgstr "Настройки Captcha Solver"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:15
msgid "&Case sensitive recognition"
msgstr "Учитывать &регистр при распознавании"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:19
msgid "Use &HTTPS"
msgstr "Использовать &HTTPS"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:23
msgid "Report image &size"
msgstr "Сообщать &размер изображения"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:27
msgid "Send &text instruction"
msgstr "Отправлять &текстовую инструкцию"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:31
msgid "Image &language:"
msgstr "&Язык изображения:"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:31
msgid "Undefined"
msgstr "Не определено"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:31
msgid "Only Cyrillic alphabet"
msgstr "Только кириллица"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:31
msgid "Only Latin alphabet"
msgstr "Только латиница"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:34
msgid "API &key:"
msgstr "&Ключ API:"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:51
msgid "&Captcha Solver Settings..."
msgstr "Настройки &Captcha Solver..."

#: ..\..\..\globalPlugins/captchaSolver/interface.py:59
msgid "Instruction text (maximum 140 characters):"
msgstr "Текст инструкции (максимум 140 символов):"

#: ..\..\..\globalPlugins/captchaSolver/interface.py:59
msgid "Sending text instruction"
msgstr "Отправка текстовой инструкции"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:6
msgid "Captcha off screen"
msgstr "Капча вне экрана"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:7
msgid "Captcha has no location"
msgstr "Капча не имеет местоположения"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:8
msgid "Error connecting to server. Please check your Internet connection"
msgstr ""
"Ошибка при подключении к серверу. Пожалуйста, проверьте ваше Интернет-"
"соединение"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:9
msgid "API key is not specified"
msgstr "Не указан ключ API"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:10
msgid "Used a non-existent API key"
msgstr "Используется несуществующий ключ API"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:11
msgid "The balance of your account is zero"
msgstr "Баланс вашего аккаунта нулевой"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:12
msgid ""
"The current recognition rate is higher than the maximum set in the settings "
"of Your account. Either on the server queue builds up and employees do not "
"have time to disassemble it, repeat the sending captcha after 5 seconds"
msgstr ""
"Текущая ставка распознования выше, чем максимально установленная в "
"настройках Вашего аккаунта. Либо на сервере скопилась очередь и работники не "
"успевают её разобрать, повторите отправку капчи через 5 секунд"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:13
msgid "Size of the captcha is less than 100 bytes"
msgstr "Размер капчи составляет менее 100 байт"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:14
msgid "Size of the captcha more than 100 KB"
msgstr "Размер капчи больше чем 100 килобайт"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:15
msgid ""
"In Your account you have configured restrictions based on IP from which you "
"can make requests. And the IP from which the request is not included in the "
"allowed list"
msgstr ""
"В Вашем аккаунте настроено ограничения по IP с которых можно делать запросы. "
"И IP, с которого пришёл данный запрос не входит в список разрешённых"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:16
msgid ""
"IP address from which the request is blocked because of frequent requests "
"with various incorrect API keys. The lock is released in an hour"
msgstr ""
"IP-адрес, с которого пришёл запрос заблокирован из-за частых обращений с "
"различными неверными ключами API. Блокировка снимается через час"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:17
msgid ""
"Captcha could not solve 3 different employee. Money for this image come back "
"to balance"
msgstr ""
"Капчу не смогли решить 3 разных работника. Деньги за это изображение "
"возвращаются обратно на баланс"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:18
msgid ""
"The error appears when 100 percent recognition. Has been used the maximum "
"number of attempts, but the required number of identical answers has not "
"been received"
msgstr ""
"Ошибка появляется при включённом 100-процентном распознании. Было "
"использовано максимальное количество попыток, но необходимое количество "
"одинаковых ответов не было набрано"

#: ..\..\..\globalPlugins/captchaSolver/responses.py:19
msgid "This captcha can not be recognized"
msgstr "Эту капчу невозможно распознать"

#~ msgid "Size image report"
#~ msgstr "Сообщать размер изображения"

#~ msgid "Text instruction"
#~ msgstr "Текстовая инструкция"

#~ msgid "To start solving captcha"
#~ msgstr "Начать решение капчи"

#~ msgid ""
#~ "I can not get the recognition result. Please check your internet "
#~ "connection"
#~ msgstr ""
#~ "Не удалось получить результат распознавания. Пожалуйста, проверьте ваше "
#~ "Интернет-соединение"

#~ msgid "Failed to get account balance. Please check your internet connection"
#~ msgstr ""
#~ "Не удалось получить баланс аккаунта. Пожалуйста, проверьте ваше интернет-"
#~ "соединение"

#~ msgid "This is not a captcha!"
#~ msgstr "Это не капча!"

#~ msgid "Recognition register has the significance"
#~ msgstr "Регистр распознавания имеет значение"

#~ msgid "Error"
#~ msgstr "Ошибка"
